kiwi/storage/subformat/vmdk.py

```diff
@@ -188,7 +188,7 @@ def _create_vmware_settings_file(self):
             with open(settings_file, 'w') as config:
                 config.write(settings_template.substitute(template_record))
                 for custom_entry in custom_entries:
-                    config.write(custom_entry)
+                    config.write(custom_entry + '\n')
```

Comment:
I think this is not safe, actually this was my first attempt, but after reading the docs and googling a little I switched to '\n'. `open('settings_file', 'w')` opens the file in text mode which, according to the docs, it is converted automatically to the platform newline character. This is some sort of write() black magic. Note that if using 'os.linesep' in windows it would fail, sinse '\r\n' would be translated to '\r\r\n'. From [here](https://docs.python.org/2/library/functions.html#open):

>The default is to use text mode, which may convert '\n' characters to a platform-specific representation on writing and back on reading.

I do not think we might having an issue because of that, however I think the right way to write a line is using '\n'. I would prefer os.linesep also, but looks like write() is doing some black magic behind the scenes :sweat: